modules/installation/proc-rhdh-deploy-eks-helm.adoc

```diff
@@ -1,46 +1,36 @@
-// Module included in the following assemblies:
-//
-// * assemblies/assembly-rhdh-integration-aws.adoc
-//
+// Module included in the following assemblies
+// assembly-install-rhdh-eks.adoc
 
-[id="proc-rhdh-deploy-eks-using-helm_{context}"]
-= Deploying {product} on {eks-name} ({eks-short}) using Helm Chart
+[id='proc-rhdh-deploy-eks-helm_{context}']
+= Installing {product-short} on {eks-short} with the Helm chart
```

Comment:
[SME] can you confirm that this change is still technically accurate?

Reply:
Yes, this looks good